src/app/components/profile/profile.component.css

```diff
@@ -21,10 +22,6 @@
     
 } 
 
-.btn-group-vertical{
```

Comment:
Are changes in this file needed? Do they resolve a code smell? They seem like minor styling changes.